testcafe/tests/pages/photos-model.js

```diff
@@ -48,21 +36,48 @@ export default class Page {
     this.modalDelete = Selector('[class*="c-modal"]').find('div')
     this.modalDeleteBtnDelete = this.modalDelete.find('button').nth(2) //REMOVE
 
-    this.allPhotos = Selector('[class*="pho-photo"]').find(
-      '[class*="pho-photo-item"]'
+    this.allPhotosWrapper = this.photoSection.find('[class^="pho-photo"]')
+    this.allPhotos = Selector('img[class*="pho-photo-item"]')
+
+    // Photo fullscreen
+    this.photoFull = Selector('[class*="pho-viewer-imageviewer"]').find('img')
+    this.photoNavNext = Selector('[class*="pho-viewer-nav--next"]')
+    this.photoNavNextBtn = this.photoNavNext.find(
+      '[class*="pho-viewer-nav-arrow"]'
+    )
+    this.photoNavPrevious = Selector('[class*="pho-viewer-nav--previous"]')
+    this.photoNavPreviousBtn = this.photoNavPrevious.find(
+      '[class*="pho-viewer-nav-arrow"]'
     )
+    this.photoBtnClose = Selector('[class*="pho-viewer-toolbar-close"]').find(
+      '[class*="c-btn"]'
+    )
+  }
+
+  async waitForLoading() {
+    await t.expect(this.loading.exists).notOk('Page still loading')
   }
 
   async getPhotosCount(when) {
     //@param {string} when : text for console.log
-    // wait until the 'exists' property of a selector is stable to count it
-    await wait(1000)
-      .until(this.allPhotos)
-      .exists.isStable()
-    const allPhotosCount = await this.allPhotos.count
-    console.log(
-      'Number of pictures on page (' + when + ' test):  ' + allPhotosCount
-    )
+
+    await this.waitForLoading()
```

Comment:
Much more better! 

Now can we create an `init()` method where you put this call too waitForLoading() and the call to getPhotosCount()?